Dockerfile

```diff
@@ -14,8 +13,7 @@ RUN \
   fi; \
   GOOS=${TARGETOS} GOARCH=${TARGETARCH} CGO_ENABLED=0 go build -v ./cmd/golink
 
-
-FROM cgr.dev/chainguard/static:latest
+FROM gcr.io/distroless/static-debian12:latest
```

Comment:
let's use `gcr.io/distroless/static-debian12:nonroot` to be consistent with what we had before.  I tested and it seems to work correctly... both map the nonroot user to uid `65532` so permissions will match ([chainguard](https://github.com/chainguard-images/images/blob/9d91ba2372bb8d33f22eab8850c79ddd39a46cdf/images/static/configs/wolfi.apko.yaml#L11), [distroless](https://github.com/GoogleContainerTools/distroless/blob/d93f2ac8cab5af40b8ea78715dde61fa3858673f/common/variables.bzl#L18))